tools/spinxsk.ps1

```diff
@@ -221,6 +221,9 @@ while (($Minutes -eq 0) -or (((Get-Date)-$StartTime).TotalMinutes -lt $Minutes))
         if ($LastExitCode -ne 0) {
             throw "SpinXsk failed with $LastExitCode"
         }
+    } catch {
+        Write-Error "Something went wrong: $_"
```

Comment:
Nit: I'd print just "Error: $_" or maybe follow the pattern here: https://github.com/microsoft/xdp-for-windows/blob/181c77437b37a8e8e0b58880e95fbb7439c82a54/tools/pktfuzz.ps1#L50-L52

I'm personally not a fan of "Something went wrong" when "Error" suffices.

Reply:
Updated.